kumascript/src/api/web.js

```diff
@@ -32,12 +32,39 @@ module.exports = {
     if (page.url) {
       if (hrefpath.toLowerCase() !== page.url.toLowerCase()) {
         if (page.url.startsWith(basepath)) {
+          let suggested = page.url.replace(basepath, "");
+          if (
+            /\//.test(suggested) &&
+            !title &&
```

Comment:
I don't think making this conditional on `!title` is strictly necessary, since it's always safe to use periods for the `domxref` macro's first argument, as they'll always be replaced when creating the `href` for the link.
https://github.com/mdn/yari/blob/a93e8d24535d8ed76c0d5d223c44cb6ffd8060e9/kumascript/macros/DOMxRef.ejs#L26
but maybe this is better since it makes the suggested argument more clear in its intended usage as the text of the link or not?

One new outcome of passing the second arg of the `domxref` macro call as the `title` arg for the `smartLink` call, is that after this PR merges, all links generated from `domxref` macro calls will now have a `title` attribute where before they never did.

Let me sleep on this, and approve tomorrow.